include/uapi/sound/sof/tokens.h

```diff
@@ -51,6 +51,7 @@
 #define SOF_TKN_SCHED_CORE			203
 #define SOF_TKN_SCHED_FRAMES			204
 #define SOF_TKN_SCHED_TIME_DOMAIN		205
+#define SOF_TKN_SCHED_DYNAMIC			206
```

Comment:
The naming is confusing. It could mean that we can change the TIME_DOMAIN dynamically, or change the scheduling parameters.

SOF_TKN_SCHED_WHEN_USED ?

Reply:
LAZY_LOAD, SETUP_ON_DEMAND ?